### ISSUE TITLE
Loading other language affects hyphenation of main language

### ISSUE THREAD

**Issue by wehro** [reutenauer/polyglossia#307]
```
\documentclass{article}
\usepackage{polyglossia}
\setmainlanguage{english}
\setotherlanguage{german}

\begin{document}
\showhyphens{directly}
\end{document}
```
The `\showhyphens` command yields `di-rect-ly`, but `di-rectly` when German is not loaded.

**Comment by eg9** [comment]
What versions? I always get `di-rectly` with or without German, either XeLaTeX or LuaLaTeX.
I'm running an up-to-date TeX Live 2019.

**Comment by wehro** [comment]
The version is the current development version of this repository.

**Comment by dohyunkim** [comment]
Counters `\l_tmpa_int` and `\l_tmpb_int` that were set during german language setting have affected the arguments of `\setlocalhyphenmins{english}` command.  
So, line 282--288 in polyglossia.sty:
```
  % setup hypenmins
  \exp_args:Nne \clist_set:Nn{\l_tmpa_clist}{\prop_item:Nn \polyglossia@langsetup {#1 / hyphenmins}}
  \int_set:Nn{\l_tmpa_int}{\clist_item:Nn{\l_tmpa_clist}{1}}
  \int_set:Nn{\l_tmpb_int}{\clist_item:Nn{\l_tmpa_clist}{2}}
  \cs_if_eq:cNF{l@#1}{\l@nohyphenation}{%
    \setlocalhyphenmins{#1}{\l_tmpa_int}{\l_tmpb_int}
  }%
```
should be changed to
```
  % setup hypenmins
  \exp_args:Nne \clist_set:Nn{\l_tmpa_clist}{\prop_item:Nn \polyglossia@langsetup {#1 / hyphenmins}}
  \cs_if_eq:cNF {l@#1} \l@nohyphenation
    {
      \use:x
        {
          \exp_not:N \setlocalhyphenmins {#1}
            { \clist_item:Nn \l_tmpa_clist {1} }
            { \clist_item:Nn \l_tmpa_clist {2} }
        }
    }
```

Incidentally, as for counter settings at line 359f:
```
      \int_set:Nn{\l_tmpa_int}{\clist_item:Nn{\l_tmpa_clist}{1}}
      \int_set:Nn{\l_tmpb_int}{\clist_item:Nn{\l_tmpa_clist}{2}}
```
these two lines, though harmless, seem to be unnecessary.

**Comment by jspitz** [comment]
@dohyunkim could you prepare a pull request with the fixes?

**Comment by jspitz** [comment]
Fixed via #308
